planner_cspace/src/planner_3d.cpp

```diff
@@ -394,6 +412,128 @@ class Planner3dNode
 
     return true;
   }
+  bool cbMakePlan(nav_msgs::GetPlan::Request &req,
+                  nav_msgs::GetPlan::Response &res)
+  {
+    if (!has_map_)
+      return false;
+
+    Astar::Vec s, e;
+    metric2Grid(s[0], s[1], s[2],
+                req.start.pose.position.x, req.start.pose.position.y, tf::getYaw(req.start.pose.orientation));
+    s[2] = 0;
+    metric2Grid(e[0], e[1], e[2],
+                req.goal.pose.position.x, req.goal.pose.position.y, tf::getYaw(req.goal.pose.orientation));
+    e[2] = 0;
+
+    if (!(cm_rough_.validate(s) && cm_rough_.validate(e)))
+    {
```

Comment:
`nav_msgs::GetPlan::Request` has `tolerance` field to relax start and goal location.
I think it's ok at now to leave it unused as is.